code/modules/urist/gamemodes/scom/civilian_mobs.dm

```diff
@@ -45,6 +45,12 @@
 	icon_living = "gunman"
 	icon_dead = "gunman_dead"
 
+/mob/living/simple_animal/hostile/scom/civ/combat/police/Initialize()
+	. = ..()
+	var/area/A = get_area(src)
+	if(A?.type == /area/spacestations/nanotrasenspace)
```

Comment:
Same for this actually. We shouldn't be checking specific areas like this, especially in this way, which will cause compile errors on glloydstation and any map that doesn't load that awaymap.